src/civis/resources/_client_pyi.py

```diff
@@ -79,16 +79,25 @@ def generate_client_pyi(client_pyi_path, api_spec_path):
                         )
                 params = inspect.signature(method).parameters
                 method_def += f"    def {method_name}(\n"
+                asterisk_added = False
                 for param_name, param in params.items():
                     annotation = _get_annotation(param)
                     if param_name == "self":
                         method_def += "        self,\n"
                     elif param.kind == inspect.Parameter.POSITIONAL_OR_KEYWORD:
                         method_def += f"        {param_name}: {annotation},\n"
                     else:
+                        if (
+                            not asterisk_added
+                            and param.kind == inspect.Parameter.KEYWORD_ONLY
+                        ):
+                            method_def += "        *,\n"
+                            asterisk_added = True
```

Comment:
Fixing a bug in the type hints for API methods that involve keyword-only arguments, which syntactically are marked by an asterisk before them in the function signature.